src/components/MainContent.js

```diff
@@ -0,0 +1,33 @@
+import handleMovie from '../services/handleMovie';
+import Button from './Button';
+
+function MainContent(initialPage) {
+    let page = initialPage;
+    const mainElement = document.createElement('main');
+    mainElement.id = 'main';
+
+    mainElement.innerHTML = `
+        <section class="item-view">
+            <h2>지금 인기 있는 영화</h2>
+            <ul class="item-list"></ul>
+        </section>
+    `;
```

Comment:
```suggestion
    const mainElement = html(`
      <main id="main">
        <section class="item-view">
            <h2>지금 인기 있는 영화</h2>
            <ul class="item-list"></ul>
        </section>
      <main>
    `);
```
이렇게 표현할 수 있도록 함수를 하나 만들어주시면 어떨까요!?